satpy/modifiers/angles.py

```diff
@@ -317,6 +317,25 @@ def _geo_dask_to_data_array(arr: da.Array) -> xr.DataArray:
     return xr.DataArray(arr, dims=('y', 'x'))
 
 
+def compute_relative_azimuth(sat_azi: xr.DataArray, sun_azi: xr.DataArray) -> xr.DataArray:
+    """Compute the relative azimuth angle.
+
+    Args:
+        sat_azi: DataArray for the satellite azimuth angles, typically in 0-360 degree range.
+        sun_azi: DataArray for the solar azimuth angles, should be in same range as sat_azi.
+    Returns:
+        A DataArray containing the relative azimuth angle in the 0-180 degree range.
+
+    NOTE: Relative azimuth is defined such that:
+    Relative azimuth is 0 when sun and satellite are aligned on one side of a pixel (back scatter).
+    Relative azimuth is 180 when sun and satellite are directly opposite each other (forward scatter).
+    """
+    ssadiff = da.absolute(sun_azi - sat_azi)
+    ssadiff = da.minimum(ssadiff, 360 - ssadiff)
```

Comment:
Just curious, can you verify that the result of this is still a `DataArray`? Also, would it be possible to switch to `np.` and still not compute things?

Reply:
Sorry, I don't follow - are you asking if we can use `np.absolute` and `np.minimum` instead? If so...maybe? I'm using dask here as I just moved the existing code to a new place!

Reply:
Yes, exactly. It cleans up the code and makes it easier to transition things in the future. Might be worth a try. If you do the check that the result is a `DataArray` and the `.data` property is a dask array then it should all be good.

Reply:
OK, I have done this now. `type(ssadiff)` is a `DataArray` and `type(ssadiff.data)` is a `dask.array.core.Array`